README.md

```diff
@@ -42,8 +42,23 @@ Note: 'localhost' as 'matrixserver' (in cfg.json) wouldn't work because of docke
 
 # Get started
 1. Create a bot user.
-2. Adjust the config file (cfg.json) to make it work with your matrix server.
-3. Invite your bot into a private room, it will join automatically.<br>
+2. Get an access token to your Matrix-Server: 
+`curl -XPOST -d '{"type":"m.login.password", "user":"@mailBotUsername:your-domain.com", "password":"mailbotPassword"}' "https://matrix.your-domain.com/_matrix/client/r0/login"`
```

Comment:
Could you make this a multi line code block? Looks better in my opinion.

Reply:
yes, you are right